src/data/__tests__/update.test.ts

```diff
@@ -37,7 +37,7 @@ describe('update', () => {
           indices: [15, 30, 25],
           values: [0.5, 0.5, 0.2],
         },
-        setMetadata: { genre: 'ambient' },
+        metadata: { genre: 'ambient' },
```

Comment:
I think this part of the test should not change, since we're still sending the same request to the backend API.